db/db.go

```diff
@@ -47,6 +47,11 @@ func (d *Database) Close() error {
 	return d.store.Close()
 }
 
+func (d *Database) TriggerGC() error {
+	// TODO: is 0.5 the right value here?
+	return d.store.Badger().RunValueLogGC(0.1)
+}
```

Comment:
will remove this comment :) and probably bump the value back up to the recommended 0.5.